.github/CODEOWNERS

```
@@ -17,6 +17,9 @@
 /stacks/dotnet80/       @msivasubramaniaan @devfile/che-team
 /stacks/dotnetcore31/   @kadel @devfile/che-team
 
+# Camel stacks
+/stacks/camel/     @brunoNetId @devfile/che-team
```

Let's omit your username for now and use the devfile services team, after the trial period we'll replace with your username:
```suggestion
/stacks/camel/ @devfile/devfile-services-team @devfile/che-team
```

---

Sorry @michael-valdron 
I don't know what I was doing... I committed a new change with your suggestion